WareDrop_Frontend/waredrop_frontend/src/dbteszt.tsx

```diff
@@ -0,0 +1,10 @@
+interface TestButtonProps {
```

Comment:
rossz elnevezés, ez egy LoginButton.
Ha pedig csak tesztelésre van és utána nem fogjátok használni, akkor most se kerüljön be a kódba